frontend/src/main.jsx

```diff
@@ -13,14 +12,15 @@ import LogHistory from "./pages/LogHistory.jsx";
 import UploadPhotos from "./pages/UploadPhoto.jsx";
 import ManualEdit from "./pages/ManualEdit.jsx";
 import "./index.css";
+import LogCode from "./pages/LogCode.jsx";
 
 createRoot(document.getElementById("root")).render(
   <AuthProvider>
     <BrowserRouter>
       <Routes>
         <Route path="/login" element={<Login />} />
         <Route path="/homepage" element={<Home />} />
-        <Route path="/signup" element={<SignUp />} />
```

Comment:
We need the signup page to redirect to the login page. Currently, all signup buttons do not work with the route changes. Alternatively could change signup button to redirect to login page.